### PR TITLE
[DM-2784] [UI] Search results from OPC UA Address space can overflow the container

### PR DESCRIPTION
**Pull request by catarino-sag**

### CODE REVIEW

File: content/change-logs/device-management/cumulocity-10-20-0-0-fixed-opc-ua-address-space-device-tab-search-results-layout.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+product_area: Device management & connectivity
+change_type:
+  - value: change-VSkj2iV9m
+    label: Fix
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/cumulocity-10-20-0-0-fixed-opc-ua-address-space-device-tab-search-results-layout.md:13]:
```suggestion
  - value: tc-pjJiURv9Y
    label: ui-c8y
```

File: content/change-logs/device-management/cumulocity-10-20-0-0-fixed-opc-ua-address-space-device-tab-search-results-layout.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+ticket: DM-2784
+version: 1020.0.0
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/cumulocity-10-20-0-0-fixed-opc-ua-address-space-device-tab-search-results-layout.md:15]: Is it agreed to not use exact version?

**Comment by catarino-sag** [comment]
@beate @MWindel I need to fix this PR. I'll update on progress.

File: content/change-logs/device-management/cumulocity-10-20-0-0-fixed-opc-ua-address-space-device-tab-search-results-layout.md (hunk 2 above)
Comment (catarino-sag) [reply content/change-logs/device-management/cumulocity-10-20-0-0-fixed-opc-ua-address-space-device-tab-search-results-layout.md]: Fixed

File: content/change-logs/device-management/cumulocity-10-20-0-0-fixed-opc-ua-address-space-device-tab-search-results-layout.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-2784
+version: 1019.19.4
+---
+Fixed OPC UA Address space device tab search results layout.
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-20-0-0-fixed-opc-ua-address-space-device-tab-search-results-layout.md:17]: OPC UA Address space device tab > what does that exactly mean? Can you please provide a screenshot?

Comment (catarino-sag) [reply content/change-logs/device-management/cumulocity-10-20-0-0-fixed-opc-ua-address-space-device-tab-search-results-layout.md:17]: Sure, there a video here: https://github.softwareag.com/IOTA/cumulocity-ui/pull/5882#issue-275182

Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-20-0-0-fixed-opc-ua-address-space-device-tab-search-results-layout.md:17]:
```suggestion
The layout of the search results in the **Address space** tab of an OPCUA device has been fixed.
```

Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-20-0-0-fixed-opc-ua-address-space-device-tab-search-results-layout.md:17]:
```suggestion
Proper search result layout on **Address space** tab of OPC UA devices
```

**Comment by catarino-sag** [comment]
@BeateRixen changes from #1918 included.

File: content/change-logs/device-management/cumulocity-10-20-0-0-fixed-opc-ua-address-space-device-tab-search-results-layout.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-2784
+version: 1019.19.4
+---
+Previously, when performing a search on small screen sizes, the search results would overflow the available space and extend beyond the visible area, making it difficult for users to view and interact with the results. This change addresses the overflow issue by ensuring the search results are properly contained within the available space on small screens. Users will now be able to easily view and navigate through the search results without any content being cut off, regardless of the screen size they are using.
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-20-0-0-fixed-opc-ua-address-space-device-tab-search-results-layout.md:17]:
```suggestion
Previously, when performing a search in the **Address space** tab of OPC UA devices on small screen sizes, the search results would overflow the available space and extend beyond the visible area, making it difficult for users to view and interact with the results. This change addresses the overflow issue by ensuring the search results are properly contained within the available space on small screens. Users will now be able to easily view and navigate through the search results without any content being cut off, regardless of the screen size they are using.
```

File: content/change-logs/device-management/cumulocity-10-20-0-0-fixed-opc-ua-address-space-device-tab-search-results-layout.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Fixed search results overflow on small screens
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-20-0-0-fixed-opc-ua-address-space-device-tab-search-results-layout.md:3]:
```suggestion
title: Search results on OPC UA Address space tab no longer overflow on small screens
```

File: content/change-logs/device-management/cumulocity-10-20-0-0-fixed-opc-ua-address-space-device-tab-search-results-layout.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Search results on OPC UA Address space tab no longer overflow on small screens
+product_area: Device management & connectivity
+change_type:
+  - value: change-VSkj2iV9m
+    label: Fix
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-20-0-0-fixed-opc-ua-address-space-device-tab-search-results-layout.md:10]:
```suggestion
    label: OPC UA
```

File: content/change-logs/device-management/cumulocity-10-20-0-0-fixed-opc-ua-address-space-device-tab-search-results-layout.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Search results on OPC UA Address space tab no longer overflow on small screens
+product_area: Device management & connectivity
+change_type:
+  - value: change-VSkj2iV9m
+    label: Fix
+component:
+  - value: component--KIsStyzM
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-20-0-0-fixed-opc-ua-address-space-device-tab-search-results-layout.md:9]:
```suggestion
  - value: component-Tf05_KQ-B 
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**